packages/solo/src/entrypoint.cjs

```diff
@@ -10,6 +10,9 @@ require('node-lmdb');
 // (dep chain: temp->glob->minimatch->brace-expansion)
 esmRequire('@agoric/install-metering-and-ses');
 
+// Expose the gc() global function, regardless of Node.js arguments.
```

Comment:
```suggestion
// Expose the gc() global function, regardless of Node.js arguments.
// WARNING: this is provided to swingset ambiently, not explicitly.
// TODO(#2160): once swingset is refactored, provide it explicitly
```

Reply:
Due to positive peer pressure, I went with the explicit provision of a non-global gc function.